src/components/ProtectedRoute/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { Route } from "react-router-dom"
+import AccessDenied from 'components/AccessDenied'
+
+const ProtectedRoute = ({ component: Component, user, ...rest }) => {
+  return (
+    <Route
+      {...rest}
+      render={(props) => {
+        if (user) {
```

Comment:
user is basically a boolean value here. The boolean values should be distinguished from normal objects. eg `user` variable should refer to the user object. isUserLoggedIn should refer to a boolean value that determines if the user is logged in or not